setup.cfg

```diff
@@ -29,7 +29,7 @@ scripts =
     bin/gnocchi-api
 
 install_requires =
-    numpy>=1.14.0
+    numpy>=2.0
```

Comment:
might be better to keep this as 1.14  to minimise requirement  conflicts depending how people install this. pp.nan exists back then too.

Reply:
done